app/src/main/res/values-es/strings.xml

```diff
@@ -1,98 +1,180 @@
     <string name="error_password_invalid">La contraseña contiene caracteres inválidos.</string>
     <string name="error_vnc_password_invalid">La contraseña de VNC contiene caracteres inválidos.</string>
-    <string name="error_vnc_password_length_incorrect">¡La contraseña de VNC debe tener entre 6 y 8 caracteres!</string>
-    <string name="services_notification_channel_description">Servicios persistentes</string>
+
+    <!-- Illegal State Dialog -->
+    <string name="illegal_state_message">%1$s \nPor favor, reinicia la app y contacta a un desarrollador.</string>
+    <string name="illegal_state_title">¡UserLAnd ha entrado en un estado ilegal!</string>
+
+    <string name="illegal_state_transition">Mala transición de estado: %1$s</string>
+
+    <string name="illegal_state_too_many_selections_when_permissions_granted">Tanto una sesión como una aplicación fueron seleccionadas cuando los permisos fueron concedidos.</string>
+    <string name="illegal_state_no_selections_when_permissions_granted">Ni una sesión ni una aplicación fueron seleccionadas cuando los permisos fueron concedidos.</string>
+
+    <string name="illegal_state_no_filesystem_selected_when_credentials_selected">Las credenciales fueron guardadas para un sistema de archivos, pero ningún sistema de archivos fue seleccionado.</string>
+    <string name="illegal_state_no_app_selected_when_preference_submitted">Los ajustes de la Aplicación fueron guardados, pero ninguna app fue seleccionada.</string>
+    <string name="illegal_state_no_app_selected_when_preparation_started">Intentando iniciar la preparación de la aplicación, pero ninguna app fue seleccionada.</string>
+    <string name="illegal_state_error_fetching_app_database_entries">No se pudo obtener acceso a la base de datos de las aplicaciones.</string> <!-- I'm not sure if this translation is correct. -->
+    <string name="illegal_state_error_copying_app_script">No se pudo copiar el script de la aplicación.</string>
+
+    <string name="illegal_state_no_session_selected_when_preparation_started">Está intentando iniciar la preparación de la sesión sin haber seleccionado una anteriormente.</string>
+    <string name="illegal_state_error_fetching_asset_lists">Ha fallado la recuperación de la lista de recursos.</string>
+    <string name="illegal_state_downloads_did_not_complete_successfully">Las descargas han fallado debido a: %1$s.</string>
+    <string name="illegal_state_failed_to_copy_assets_to_local">Ha fallado la copia de los recursos al almacenamiento local.</string>
+    <string name="illegal_state_failed_to_copy_assets_to_filesystem">Ha fallado la copia de los recursos al sistema de archivos.</string>
+    <string name="illegal_state_failed_to_extract_filesystem">Ha fallado la extracción del sistema de archivos.</string>
+    <string name="illegal_state_filesystem_is_missing_assets">Al sistema de archivos le faltan recursos. Prueba a borrar los archivos de soporte.</string>
```

Comment:
We can remove line 69 (`illegal_state_filesystem_is_missing_assets`) since the English strings.xml doesn't have this entry anymore.